manager/state/raft/raft_test.go

```diff
@@ -260,6 +263,7 @@ func TestRaftFollowerLeave(t *testing.T) {
 	assert.NoError(t, err, "error sending message to leave the raft")
 	assert.NotNil(t, resp, "leave response message is nil")
 
+	raftutils.ShutdownNode(nodes[5])
```

Comment:
All util functions receive map of nodes. This node is removed from map if I understand code correctly